### PR TITLE
feat(display_name): Auto detector for display_name options

### PR DESCRIPTION
**Pull request by dhyegocalota**

### CODE REVIEW

File: lib/activeadmin_addons/support/input_helpers/select_helpers.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def item_to_select_option(item)
       return unless item
       {
         id: item.send((valid_options[:value] || :id)),
-        text: item.send((valid_options[:display_name] || :name))
+        text: item.send((valid_options[:display_name] || ActiveadminAddons.default_display_name(item)))
```
Comment (monkeyci) [inline lib/activeadmin_addons/support/input_helpers/select_helpers.rb]: Line is too long. [103/100]

File: lib/activeadmin_addons.rb [hunk 2]
```diff
@@ -18,6 +18,19 @@ def datetime_picker_input_format
     @datetime_picker_input_format
   end
 
+  def display_name_methods
+    @display_name_methods ||= ActiveAdmin.application.display_name_methods
+  end
+
+  def default_display_name(object)
+    (display_name_methods - association_methods(object)).detect { |method| object.respond_to?(method) }
```
Comment (monkeyci) [inline lib/activeadmin_addons.rb]: Line is too long. [103/100]

File: lib/activeadmin_addons.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 module ActiveadminAddons
   extend self
 
-  attr_writer :default_select, :datetime_picker_default_options, :datetime_picker_input_format
+  attr_writer :default_select, :datetime_picker_default_options, :datetime_picker_input_format, :display_name_methods
```
Comment (monkeyci) [inline lib/activeadmin_addons.rb]: Line is too long. [117/100]

**Comment by jgmontoya** [comment]
I'm sorry this was left unattended for so long. If you're still interested in this, please rebase it, add some explanations as to what this solves and if possible add some tests so we can catch the problem in the future.

**Pull request closed without merging**